main.go

```diff
@@ -31,17 +31,18 @@ const (
 
 func main() {
 	cnfg := config.DefaultConfig()
-	rootCmd, globalFlagValues := rootcmd.New(&cnfg.Hazelcast)
+	rootCmd, persistentRootFlags := rootcmd.New(&cnfg.Hazelcast)
 	programArgs := os.Args[1:]
 	// update config before running root command to make sure flags are processed
-	err := updateConfigWithFlags(rootCmd, cnfg, programArgs, globalFlagValues)
+	err := updateConfigWithFlags(rootCmd, cnfg, programArgs, persistentRootFlags)
 	ExitOnError(err)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 	isInteractive := IsInteractiveCall(rootCmd, programArgs)
 	if isInteractive {
 		RunCmdInteractively(ctx, rootCmd, &cnfg.Hazelcast)
 	} else {
+		rootCmd, _ = rootcmd.New(&cnfg.Hazelcast)
```

Comment:
Why is this change necessary? If it is necessary, can you add a comment about why the second argument is ignored.

Reply:
Explicit parsing is made to get cluster config before the execution in previous steps. It changes the rootCmd properties. For not to break cobra internally, rootCmd is overwritten as in the beginning intentionally. I added comment about why second argument is ignored. [7a991c2](https://github.com/hazelcast/hazelcast-commandline-client/pull/67/commits/7a991c20b78270f67c8e763f4727f6ef1e3a3463).